pull_request_template.md

```diff
@@ -1,23 +1,34 @@
-# PULL REQUEST
-
-## Purpose and Content
-(Clearly and concisely state the purpose of this PR)
-
-## Benefits and Risks
-(State concisely the benefits to be derived from and the risks associated with this PR)
-
-## Linked Issues
-(Provide references to any link issues. Use closes #issuenum to automatically close an open issue)
-- Fixes #
-- Closes #
-
-## PR Checklist
-- [ ] This PR has a corresponding issue OR is linked to an SDI.
-- [ ] I have followed CliMA's codebase [contribution](https://clima.github.io/ClimateMachine.jl/latest/Contributing/) and [style](https://clima.github.io/ClimateMachine.jl/latest/DevDocs/CodeStyle/) guidelines OR N/A.
-- [ ] I have followed CliMA's [documentation policy](https://github.com/CliMA/policies/wiki/Documentation-Policy).
-- [ ] I have checked all issues and PRs and I certify that this PR does not duplicate an open PR.
-- [ ] Unit tests are included OR N/A.
-- [ ] Code used in an integration test OR N/A.
-- [ ] All tests ran successfully on my local machine OR N/A.
-- [ ] All classes, modules, and function contain appropriate docstrings OR N/A.
-- [ ] Documentation has been added/updated OR N/A.
+<!--- THESE LINES ARE COMMENTED -->
+## Purpose 
+<!--- One sentence to describe the purpose of this PR, refer to any linked issues:
+#14 -- this will link to issue 14
+Closes #2 -- this will automatically close issue 2 on PR merge
+-->
+
+
+## To-do
+<!---  list of proposed tasks in the PR, move to "Content" on completion 
+- Proposed task
+-->
+
+
+## Content
+<!---  specific tasks that are currently complete 
```

Comment:
Wouldn't checking off the boxes in the To-do list also mark them as complete? 
Maybe I am reading this in a way for which the difference between "Content" and the items in the To-do list is very subtle to me

Reply:
Sounds good, thanks for your explanation. It sounds more clear to me now

Reply:
(And of course you are welcome to use checkboxes too to monitor progress! but I won't go into this much detail)